src/displayapp/screens/StopWatch.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include "Screen.h"
+#include "components/datetime/DateTimeController.h"
+#include "../LittleVgl.h"
+
+#include "FreeRTOS.h"
+#include "portmacro_cmsis.h"
+
+#include <array>
+
+namespace Pinetime::Applications::Screens {
+
+  enum class States { INIT, RUNNING, HALTED };
```

Comment:
Other enum classes do not name their values with CAPITAL letters. Could you rename these as 'Init', 'Running' and 'Halted' for more coherence?
Same for Events.